src/rover/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@
 # other details. No copyright assignment is required to contribute to Ascent.
 ###############################################################################
 
+# Always build rover as a static PIC lib
+set(BUILD_SHARED_LIBS OFF)
```

Comment:
thought: Rover uses VTK-m, so if we do need to change to static libs here, I would like better understand why. This could be a signal something is missing. 

We will need to run tests with real cuda hw --  b/c even if we survive linking, we could get immediate runtime sadness.

Reply:
ah, I missed that dep. The only reason it is forced static is because it is hard-coded as librover.a in a couple [places](https://github.com/Alpine-DAV/ascent/blob/develop/src/ascent/CMakeLists.txt#L411).

Although, does it matter if rover is compiled with PIC? I would think since it is internal and has PUBLIC deps from cmake the libraries would be linked correctly in the end.

Reply:
gotcha -- I think we can adjust to look for it in the right place and use top level choice of static/shared.  I think we just need to ask cmake about the rover target.

Reply:
Okay, updated to use a generator expression to resolve the target lib location.

Reply:
Thanks!